### ISSUE TITLE
Natural abundances for Te126 and Ta180 are incorrect

### ISSUE THREAD

**Issue by paulromano** [lmmentel/mendeleev#122]
## Describe the bug

The natural abundance of Te126 as given in IUPAC 2013 is 18.84%, whereas `mendeleev` reports it to be 8.84%.

The natural abundance of Ta180 as given in IUPAC 2013 is 0.01201%, whereas `mendeleev` reports no value.

## To Reproduce
```python
from mendeleev import isotope
print(isotope(52, 126).abundance)
print(isotope(73, 180).abundance)
```

## Expected behavior

`isotope(52, 126).abundance` should give 18.84, and `isotope(73, 180).abundance` should give 0.01201.

## Specification

- mendeleev master branch
- python version 3.10.6
- Ubuntu 20.04

**Comment by lmmentel** [comment]
Good catch!

Do you have a data source for the values you are mentioning. I took the isotope data from [this file](https://www-nds.iaea.org/amdc/ame2020/nubase_3.mas20.txt) provided by [Nuclear Data Services](https://www-nds.iaea.org/).

I checked again and for 126Tc the abundance (given in the IS= field) is 8.84, here's the actual data from the source file:

```
126 0520   126Te  -90064.2        1.4                                 stbl              0+            03          1924 IS=8.84 25
```

I also checked the abundance for 180Ta and indeed it's missing from this source (missing IS= field):
```
180 0730   180Ta  -48933.6        2.1                                   8.154  h 0.006  1+            15          1938 EC=85 3;B-=15 3
```

It could be that the data in this source is outdated or contains errors but I would like to investigate the source you mention first to compare. Could you share where the values are coming from?

**Comment by lmmentel** [comment]
Hey I compared the source you provided with the original and indeed the only change made is abundance for 126Te. Should be fixed in #123 . Thanks again!
